src/plugins/irc/irc-config.c

```diff
@@ -1712,6 +1712,18 @@ irc_config_server_new_option (struct t_config_file *config_file,
                 callback_change, callback_change_data,
                 NULL, NULL);
             break;
+        case IRC_SERVER_OPTION_SSL_CA_FILE:
+            new_option = weechat_config_new_option (
+                config_file, section,
+                option_name, "string",
+                N_("File containing the certificate authorities. "),
```

Comment:
For consistency this option description should be similar to the others, i.e. start lowercase and no full stop.